# oncoanalyser: tool descriptions

## WiGiTS tools

| Name                                                                                         | Category               | Function                                                        |
| -                                                                                            | -                      | -                                                               |
| [AMBER](https://github.com/hartwigmedical/hmftools/tree/master/amber)                        | Copy-number variants   | Calculate tumor Î²-allele frequencies for PURPLE copy number fit |
| [BamTools](https://github.com/hartwigmedical/hmftools/tree/master/bam-tools)                 | Alignment processing   | Generate DNA alignment statistics                               |
| [CHORD](https://github.com/hartwigmedical/hmftools/tree/master/chord)                        | HRD                    | Predict homologous recombination deficiency                     |
| [COBALT](https://github.com/hartwigmedical/hmftools/tree/master/cobalt)                      | Copy-number variants   | Calculate read depth ratios for PURPLE copy number fit          |
| [CUPPA](https://github.com/hartwigmedical/hmftools/tree/master/cuppa)                        | Tissue of origin       | Predict tumor tissue of origin                                  |
| [ESVEE](https://github.com/hartwigmedical/hmftools/tree/master/esvee)                        | Structural variants    | Call structural variants                                        |
| [ISOFOX](https://github.com/hartwigmedical/hmftools/tree/master/isofox)                      | Transcript processing  | Characterise, count gene and transcript features                |
| [LILAC](https://github.com/hartwigmedical/hmftools/tree/master/lilac)                        | HLA characterisation   | Type HLA-alleles and annotate with variants                     |
| [LINX](https://github.com/hartwigmedical/hmftools/tree/master/linx)                          | Structural variants    | Annotate, interpret, and visualise structural variants          |
| [Neo](https://github.com/hartwigmedical/hmftools/tree/master/neo)                            | Neo-epitope prediction | Predict and rank neo-epitopes                                   |
| [PAVE](https://github.com/hartwigmedical/hmftools/tree/master/pave)                          | Small variants         | Annotate small variants with gene, transcript, protein impacts  |
| [PURPLE](https://github.com/hartwigmedical/hmftools/tree/master/purple)                      | Copy-number variants   | Fit purity/ploidy model and annotate variants with CN data      |
| [REDUX](https://github.com/hartwigmedical/hmftools/tree/master/redux)                        | Alignment processing   | Process DNA alignments (control artefacts, deduplicate, unmap)  |
| [SAGE](https://github.com/hartwigmedical/hmftools/tree/master/sage)                          | Small variants         | Call small variants                                             |
| [SAGE append](https://github.com/hartwigmedical/hmftools/tree/master/sage)                   | Small variants         | Append evidence (e.g. RNA support) to small variants            |
| [Sigs](https://github.com/hartwigmedical/hmftools/tree/master/sigs)                          | Mutational signatures  | Calculate mutational signatures                                 |
| [VirusInterpreter](https://github.com/hartwigmedical/hmftools/tree/master/virus-interpreter) | Viral detection        | Filters and annotates VIRUSBreakend calls                       |

## Other tools

| Name                                                                                        | Category             | Function              |
| -                                                                                           | -                    | -                     |
| [BWA-MEM2](https://github.com/bwa-mem2/bwa-mem2)                                            | Read alignment       | DNA read alignment    |
| [GATK MarkDuplicates](https://github.com/broadinstitute/gatk)                               | Alignment processing | RNA duplicate marking |
| [STAR](https://github.com/alexdobin/STAR)                                                   | Read alignment       | RNA read alignment    |
| [VIRUSBreakend](https://github.com/PapenfussLab/gridss/blob/master/VIRUSBreakend_Readme.md) | Viral detection      | Viral detection       |
